src/error/TryCatchHandler.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+type ErrorHandler<T> = (error: Error) => T | undefined;
+
+/**
+ * Try to execute the code block and catch the error if it is of the given type.
+ * If the error is of the given type, the fallback value or the result of the function is returned.
+ * If the function returns undefined or the error type do not match, the error is normally thrown and not handled.
+ *
+ * @param codeBlock Code block to execute
+ * @param errorType Errortype to catch and handle
+ * @param fallbackValueOrFunction Fallback value or function to compute the fallback value
+ */
+export function tryCatch<T>(codeBlock: () => T, errorType: {new (...args: any[]): Error}, fallbackValueOrFunction: ErrorHandler<T> | T): T {
+    try {
```

Comment:
I really like the simplicity of not having to define a function if you have a "static rturn value" as you proposed earlier ... So the function was just because there are also other cases. So I would live the complexity her ein this one place to have more simple code in all other places ;-)